i6_native_ops/monotonic_rnnt/__init__.py

```diff
@@ -0,0 +1,237 @@
+import os
+from typing import Optional
+
+import torch
+from pkg_resources import get_distribution
+from torch.utils.cpp_extension import load
+
+current_file_path = os.path.abspath(__file__)
+current_directory = os.path.dirname(current_file_path)
+
+try:
+    # Package is installed, so ops are already compiled
+    __version__ = get_distribution("i6_native_ops").version
+    import i6_native_ops.monotonic_rnnt.monotonic_rnnt_core as core
+except Exception as e:
+    # otherwise try to build locally
+    from torch.utils.cpp_extension import load
+
+    base_path = os.path.dirname(__file__)
+    core = load(
+        name="monotonic_rnnt_core",
+        sources=[
+            f"{base_path}/monotonic_rnnt.cu",
+        ],
+        extra_cuda_cflags=["-DRNNT_ENABLE_GPU"],
+        extra_include_paths=[f"{base_path}/include/"],
+        verbose=True,
+    )
+
+
+class MonotonicRNNTFunction(torch.autograd.Function):
+    @staticmethod
+    def forward(
+        ctx,
+        acts: torch.Tensor,
+        labels: torch.Tensor,
+        input_lengths: torch.Tensor,
+        label_lengths: torch.Tensor,
+        alignment: Optional[torch.Tensor] = None,
+        max_shift_from_alignment: int = 0,
+        blank_label: int = 0,
+    ) -> torch.Tensor:
+        assert core is not None
```

Comment:
Would this ever be the case? Wouldn't `torch.utils.cpp_extension.load` just throw an error instead of returning `None`?